auslib/web/admin/views/history.py

```diff
@@ -76,6 +85,82 @@ def get_revisions(self,
         ret['count'] = total_count
         return jsonify(ret)
 
+    def get_all_revisions(self,
```

Comment:
There is almost no difference between this function and `get_revisions` - are you sure you need this? If it does have unique functionality, are you sure you need it? Can that functionality be rolled into `get_revisions`?

Reply:
Though I have changed a lot of things in entire code, this suppose to return all history without default limit of 10 when the user queries the endpoint without providing limits and pages. @andela-hngerebara requested that I should not limit the history to 10, instead she needs the entire history for has UI task.